_config.yml

```diff
@@ -1,6 +1,39 @@
-remote_theme: pmarsceill/just-the-docs
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
 
-# logo: "/assets/dc.png"
-title: "datacommons.org"
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+title: "Data Commons Docs"
+description: Data Commons Documentation
+url: "https://docs.dataacommons.org"
```

Comment:
@beets I think this typo breaks some links, e.g. the one to "Statistical Variables" on [this page](https://docs.datacommons.org/api/).

Reply:
Thanks for pointing it out. Fixed in #46